src/TestFramework/TestFramework/Resources/FrameworkMessages.resx

```diff
@@ -138,6 +138,18 @@
   <data name="AreEqualDifferentTypesFailMsg" xml:space="preserve">
     <value>Expected:&lt;{1} ({2})&gt;. Actual:&lt;{3} ({4})&gt;. {0}</value>
   </data>
+  <data name="AreEqualStringDiffFailMsg" xml:space="preserve">
+    <value>{1} {2}
+{3}
+{4}
+{5} {0}</value>
```

Comment:
The user message should be toward the top (before expected and actual values) not the end